packages/frontend/src/pages/Convert/ConvertContent.tsx

```diff
@@ -105,9 +106,16 @@ const ConvertContent: FC = () => {
     approveTokens()
   }
 
+  useEffect(() => {
+    if (sourceNetwork?.slug === ChainSlug.Polygon || destNetwork?.slug === ChainSlug.Polygon) {
+      return setManualWarning('Warning: transfers to/from Polygon are temporarily down.')
```

Comment:
is this still needed?

Reply:
this is old code. after rebasing, it's up-to-date